### PR TITLE
Added remaining card components

### PR DESCRIPTION
**Pull request by jeetbhalani15**
## Added Card With Text Only & Card With Shadow.
-Added card with text component CSS & HTML
-Added Cart with Shadow Component

image
image

### CODE REVIEW

File: Documentation/docs-comp-css/main-card.css [hunk 1]
```diff
@@ -370,6 +370,11 @@ small{
   /*
```
Comment (ckahub) [inline Documentation/docs-comp-css/main-card.css:370]: delete commented code

File: Documentation/docs-comp-css/main-card.css [hunk 2]
```diff
@@ -370,6 +370,11 @@ small{
   /* 
   cards css
   */
+  .shadow{
+      padding: 2rem;
+      border: 1px solid rgb(158, 158, 158);
+      box-shadow: 0px 0px 18px rgb(231, 216, 164);
```
Comment (ckahub) [inline Documentation/docs-comp-css/main-card.css:376]: use rem & hex for color

File: Documentation/docs-comp-css/main-card.css [hunk 3]
```diff
@@ -481,7 +486,7 @@ vertical cards
     position: relative;
     flex-direction: column;
     gap: 1rem;
-    background-color: #f1f1f1;
+    /* background-color: #f1f1f1; */
```
Comment (ckahub) [inline Documentation/docs-comp-css/main-card.css]: delete coment code

File: Documentation/docs-comp-html/main-card.html [hunk 4]
```diff
@@ -213,16 +234,16 @@ <h1>Example of vertical Card Text + Image</h1>
...
                       sandbox="allow-scripts allow-same-origin">
                     </iframe> </div>
                     <h1>Example of Cards with Text Overlay</h1>
 
                     <div class="card-vertical-box">
-                      <div class="vertical-card overlay">
+                      <div class="vertical-card card-overlay">
                       <div class="card-top">
-                          <img class="card-img" src="https://cdn1.epicgames.com/offer/3ddd6a590da64e3686042d108968a6b2/EGS_GodofWar_SantaMonicaStudio_S2_1200x1600-fbdf3cbc2980749091d52751ffabb7b7_1200x1600-fbdf3cbc2980749091d52751ffabb7b7?h=854&resize=1&w=640" alt="">
+                          <img class="card-img" src="https://cdn1.epicgames.com/b4565296c22549e4830c13bc7506642d/offer/TETRA_PREORDER_STANDARD EDITION_EPIC_Store_Portrait_1200x1600-1200x1600-ca8b802ff13813c37a44ebf68d0946a2.png?h=854&resize=1&w=640"  alt="">
```
Comment (ckahub) [inline Documentation/docs-comp-html/main-card.html:244]: give description at img alt

File: Documentation/docs-comp-css/main-card.css (hunk 1 above)
Comment (jeetbhalani15) [reply Documentation/docs-comp-css/main-card.css:370]: ok sure

**Comment by Jay-Gadhiya** [comment]
- class names are good
- code written well
- you can correct suggested changes by @ckahub

**Review by Jay-Gadhiya — APPROVED** [review]

File: Documentation/docs-comp-html/main-card.html (hunk 4 above)
Comment (jeetbhalani15) [reply Documentation/docs-comp-html/main-card.html:244]: made chnages ypu can approve it too

File: Documentation/docs-comp-css/main-card.css (hunk 1 above)
Comment (ckahub) [reply Documentation/docs-comp-css/main-card.css:370]: great

**Pull request merged**